EDU-471 cloud docs refresh branch (#2213)
temporalio/documentation: docs-src/cloud/support-intro.md

---
id: support-intro
title: How do I get support from Temporal
sidebar_label: Support

> description: We have a very active community Slack and a very active community forum where you can post questions and ask for help. We also offer ticketing system for Temporal Cloud customers.

Reviewer: ```suggestion
description: We have an active community Slack and an active community forum where you can post questions and ask for help. We also offer a ticketing system for Temporal Cloud customers.
```

tags:
  - temporal-cloud
  - support
  - introduction
---

> There is an active [community Slack](https://temporalio.slack.com) and an active [community Discourse forum](https://community.temporal.io/) where you can post questions and ask for help. We also offer ticketing system for Temporal Cloud customers.

Reviewer: ```suggestion
We have an active [community Slack](https://temporalio.slack.com) and an active [community Discourse forum](https://community.temporal.io/) where you can post questions and ask for help.
We also offer a ticketing system for Temporal Cloud customers.
```
